test.ts

```diff
@@ -843,17 +829,17 @@ var TESTS: Test[] = [
         delimiter: '/',
         optional: true,
         repeat: false,
-        partial: true,
         pattern: '[^\\/]+?'
       },
       '-bar'
     ],
     [
-      ['/-bar', ['/-bar', undefined]],
+      ['-bar', ['-bar', undefined]],
+      ['/-bar', null],
```

Comment:
This change seems contrary to what I'd expect, which is that the delimiter at the beginning of a path would be required. I think perhaps this is an example where escaping the `/` would accomplish that?

Reply:
Yes, to have this be required you would escape the slash making it a regular character instead of a prefix.